ci/authn-gcp/cleanup_function.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash -ex
+
+# Script that runs inside a Google SDK container and deploys a function
+# That fetches an id token with audience claim passed in the request query string.
```

Comment:
the description says that this script deploys a function but the script name is `cleanup_function`. please fix